src/monodroid/jni/monodroid-glue.cc

```diff
@@ -980,14 +981,14 @@ MonodroidRuntime::init_android_runtime (MonoDomain *domain, JNIEnv *env, jclass
 		lookup_bridge_info (domain, image, &osBridge.get_java_gc_bridge_type (i), &osBridge.get_java_gc_bridge_info (i));
 	}
 
-	// TODO: try looking up the method by its token
-	MonoClass *runtime = utils.monodroid_get_class_from_image (domain, image, "Android.Runtime", "JNIEnv");
-	MonoMethod *method = mono_class_get_method_from_name (runtime, "Initialize", 1);
+	// Android.Runtime.JNIEnv
+	MonoClass *runtime = utils.monodroid_get_class (domain, image, managed_token_ids.android_runtime_jnienv);
+	abort_unless (runtime != nullptr, "INTERNAL ERROR: Unable to find class Android.Runtime.JNIEnv in the Mono.Android assembly");
```

Comment:
We should prefer an existing convention when one exists.  In this case, this is an assembly-qualified name, thus:

```cc
"INTERNAL ERROR: Unable to find type `Android.Runtime.JNIEnv, Mono.Android`!"
```